### DISCUSSION TITLE
Explore using esbuild-plugin

### DISCUSSION THREAD

**Discussion by themre** [gatsbyjs/gatsby#29341] [Ideas / Feature Requests]
Hi,
I know you were probably already asked this, but didn't find any question... Since now there is all the hype about using esbuild-plugin in Vite and other tools, would it be interesting to explore adding esbuild plugin into the gatsby itself? This would definitely improve whole build and DX by a lot. 

Thanks for the answer!

**Comment by vincemtnz** [comment]
Not the answer you're looking for but whatever efforts are required to do this will be similar to the efforts required to use swc-plugin -- discussion: https://github.com/gatsbyjs/gatsby/discussions/29542
